src/main/java/com/software/modsen/ratingmicroservice/observer/DriverRatingObserver.java

```diff
@@ -22,12 +22,14 @@ public class DriverRatingObserver implements RatingObserver {
     @Retryable(retryFor = {DataAccessException.class, FeignException.class}, maxAttempts = 5,
             backoff = @Backoff(delay = 500))
     public void updateRatingSource(RatingInfo ratingInfo) {
-        if (ratingInfo.getRatingSource().equals(Source.PASSENGER)) {
+        if (ratingInfo.getRatingSource().equals(SimpleRatingSource.PASSENGER)) {
             ResponseEntity<Ride> rideFromDb = rideClient.getRideById(ratingInfo.getRating().getRide().getId());
 
             DriverRatingMessage driverRatingMessageValue = new DriverRatingMessage(rideFromDb.getBody().getDriver().getId(),
                     ratingInfo.getRating().getRatingValue());
 
+            System.out.println("\n\n\n" + driverRatingMessageValue + "\n\n\n");
```

Comment:
No System.out.printlns in production code

Reply:
Use logging instead if you want to see the value. Logging is your friend, which can be in prod